gcloud/monitoring/client.py

```diff
@@ -490,3 +490,83 @@ def list_groups(self):
         :returns: A list of group instances.
         """
         return Group._list(self)
+
+    def write_time_series(self, timeseries_list):
+        """Writes a list of time series objects to the API.
+
+        The recommended approach to creating time series objects is using
+        the :meth:`~gcloud.monitoring.client.Client.time_series` factory
+        method.
+
+        To write a single time series object, consider using the
+        :meth:`~gcloud.monitoring.client.Client.write_point` method.
+
+        Example::
+
+            >>> client.write_time_series([ts1, ts2])
```

Comment:
You are talking about writing a single time series object & write_point above, but use write_time_series in the code sample. Please make them consistent.